Bookings/BookingStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Airport_Ticket_Booking_System.Bookings;
+
+public enum BookingStatus
+{
+    Confirmed,
+    Pending,
+    Cancelled
+
+}
```

Comment:
```suggestion
public enum BookingStatus
{
    Unknown = 0,
    Confirmed = 1,
    Pending = 2,
    Cancelled = 3
}
```

It's always better to add the expected value for each state here, for better readability. Also, it's good to add an Unknown state with value = 0 to track bugs. We can discuss this more in tomorrow's meeting. 


Please apply this fix to all other enums.